### DISCUSSION TITLE
How to remove textarea?

### DISCUSSION THREAD

**Discussion by kimsanghoon1** [eclipse-theia/theia#8271] [General]
![image](https://user-images.githubusercontent.com/16382067/88902968-de4c8580-d28d-11ea-922e-9b70a0c4818f.png)
Text area is visible in the terminal.
How to remove Text area?

**Comment by vince-fugnitto** [comment]
@kimsanghoon1 if you are referring to the bug in the terminal (extra cursor-looking item), it has been fixed https://github.com/eclipse-theia/theia/pull/8204, and a new release is scheduled for later today (to update `latest`). If it is not what you are referring to in your question can you please elaborate?
